yb-voyager/cmd/common_test.go

```diff
@@ -0,0 +1,397 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
+
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/migassessment"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/testutils"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+func TestAssessmentReportStructs(t *testing.T) {
```

Comment:
similar comment about defining all the structs to tests in a struct array

Reply:
Yep done